app/controllers/users_controller.rb

```diff
@@ -0,0 +1,3 @@
+class UsersController < ApplicationController
+  def index; end
+end
```

Comment:
[REQUIRED] I noticed that a user is redirected to the splash screen after login. Kindly redirect the user to the categories page instead